static/js/mass-update.js

```diff
@@ -95,19 +102,18 @@ $(document).ready(function() {
                 }
             });
         }
+
         if (totalCount === 0) {
             return false;
         }
-        var params = $.param({
-            toUpdate: updateArr.join('|'),
-            toRefresh: refreshArr.join('|'),
-            toRename: renameArr.join('|'),
-            toSubtitle: subtitleArr.join('|'),
-            toDelete: deleteArr.join('|'),
-            toRemove: removeArr.join('|'),
-            toMetadata: metadataArr.join('|')
+        if (updateArr.length + refreshArr.length + renameArr.length + subtitleArr.length + deleteArr.length + removeArr.length + metadataArr.length === 0) {
+            return false;
+        }
+        const url = $('base').attr('href') + 'manage/massUpdate';
+        const params = 'toUpdate=' + updateArr.join('|') + '&toRefresh=' + refreshArr.join('|') + '&toRename=' + renameArr.join('|') + '&toSubtitle=' + subtitleArr.join('|') + '&toDelete=' + deleteArr.join('|') + '&toRemove=' + removeArr.join('|') + '&toMetadata=' + metadataArr.join('|');
+        $.post(url, params, function() {
+            location.reload(true);
```

Comment:
With a nonce you could still refresh and delay the response until action is completed.  A nonce would also prevent a doubled action.